recipes/init.rb

```diff
@@ -54,3 +54,20 @@
     end
   end
 end
+
+include_recipe 'krb5' if hadoop_kerberos?
```

Comment:
minor: seems like one ``if hadoop_kerberos?`` around these 3 statements would be cleaner instead of guards for each one.  unless rubocop says otherwise i guess...